tests/test_signature_handler.py

```diff
@@ -800,3 +800,45 @@ def test_sign_claim_messages_no_signatures(
     mock_get_radas_signatures.assert_not_called()
     mock_validate_radas_msgs.assert_not_called()
     mock_upload_signatures_to_pyxis.assert_not_called()
+
+
+@mock.patch("pubtools._quay.signature_handler.QuayClient")
+@mock.patch("pubtools._quay.signature_handler.QuayApiClient")
+def test_construct_item_claim_messages_invalid_signing_key(
```

Comment:
can we please rename this to on_signing_key. In the future we'll implement condition to accept only 16 characters long signing key and therefore invalid signing key will be anything shorter. So current name could lead to misunderstanding

Reply:
Yeah, updated.